### ISSUE TITLE
feature request : support uploading image to qiniu(七牛)

### ISSUE THREAD

**Issue by NanerLee** [shd101wyy/markdown-preview-enhanced#364]
atom里有上传到七牛的插件, 但是要么很久没维护了, 要么跟这个插件功能上有重复的地方.
不太愿意为了喘一个图片再多安一个插件. 所以要是本插件能支持上传到七牛就完美了.

**Comment by shd101wyy** [comment]
Hi，直接本插件 Image Helper 然后选择 sm.ms upload 就好了。

**Comment by NanerLee** [comment]
@shd101wyy 我用过这个v站站友的图床, 很不错, 但是由于没有帐号体系, 后续就无法管理这些图片, 而且如果链接忘了,还得重传一次. 所以很少使用它.

**Comment by lizyn** [comment]
Yeah, I'd appreciate more uploading choices, e.g., the popular qiniu cloud.

**Comment by youngjuning** [comment]
@shd101wyy 是啊，希望你可以引入七牛云，还有另一个图床不能用，直接删了好了

**Comment by shd101wyy** [comment]
`youngjuning` 我记得 `sm.ms` 是可以用的啊？  
我支持不了七牛，因为我在国外。我试过七牛的账号我都注册不了。。。

**Comment by youngjuning** [comment]
sm.ms可以，另一个不行

**Comment by shd101wyy** [comment]
Imgur 国外用的话比 sm.ms 快。国内是被墙了

**Comment by youngjuning** [comment]
额，看来我得自己加七牛云了，主要我还不会写atom插件
